### ISSUE TITLE
pod "Branch" is out of date

### ISSUE THREAD

**Issue by guilhermeagostinelli** [claimsforce-gmbh/flutter-segment#44]
Hi,

I'm using Segment v2.2.2 but it's conflicting with the Branch SDK itself ([flutter_branch_sdk](https://pub.dev/packages/flutter_branch_sdk) package) which I need in order to handle Branch Deep Links.

```
[!] CocoaPods could not find compatible versions for pod "Branch":
  In Podfile:
    Branch

    flutter_branch_sdk (from `.symlinks/plugins/flutter_branch_sdk/ios`) was resolved to 1.2.0, which depends on
      Branch (~> 0.36.0)

    flutter_segment (from `.symlinks/plugins/flutter_segment/ios`) was resolved to 0.0.1, which depends on
      Segment-Branch (= 0.1.22) was resolved to 0.1.22, which depends on
        Branch (~> 0.33.0)
```

Is there an alternative besides upgrading to Segment v3.0.0 and disabling the Branch integration? If I do so, will Segment be able to send the app events to Branch? I would also like to send some attribution data such as campaign info to Amplitude so I presume that if I use Segment v3.0.0 I would need to do that manually by fetching this info through the Branch SDK and then send it to Amplitude myself through its [Attribution API](https://developers.amplitude.com/docs/attribution-api), right?

Thanks in advance.

**Comment by viniciuszani** [comment]
Hi @guilhermeagostinelli, I had this issue today (also using branch plugin).

It seems that this issue is fixed in this repo's master.
While it isn't published as a new version, you can reference it directly from your pubspec: 

```yaml
  flutter_branch_sdk: ^1.3.0
  flutter_segment:
    git:
      url: git://github.com/claimsforce-gmbh/flutter-segment.git
```

Can you check if it works?

**Comment by guilhermeagostinelli** [comment]
Hi @viniciuszani , thanks for the reply.

It should work because, presumably, the master branch should have the latest released version which is currently v3.0.0. The problem with v3.0.0 is that one is not supposed to use `ENABLE_BRANCH_IO_INTEGRATION` since the Branch integration has been removed.

**Comment by guilhermeagostinelli** [comment]
Hi @viniciuszani , thanks for the reply.

It works (but so do v3.0.0). Are you sure one can use the `ENABLE_BRANCH_IO_INTEGRATION` with this version?

The last significant change in the master branch has been introduced by https://github.com/claimsforce-gmbh/flutter-segment/commit/f01f2e65d1683f628b4142ae659f6f64ffe505ab which has then removed the Branch integration so I don't think the master branch is any different than the latest released package.
